contents/docs/llm-analytics/_snippets/embedding-event.mdx

```diff
@@ -0,0 +1,53 @@
+An embedding is a single call to an embedding model to convert text into a vector representation.
+
+Event Name: `$ai_embedding`
+
+| Property | Description |
+|----------|-------------|
+| `$ai_trace_id` | The trace ID (a UUID to group related AI events together)<br/>Must contain only letters, numbers, and special characters: `-`, `_`, `~`, `.`, `@`, `(`, `)`, `!`, `'`, `:`, <code>\|</code><br/>Example: `d9222e05-8708-41b8-98ea-d4a21849e761` |
+| `$ai_model` | The embedding model used<br/>Example: `text-embedding-3-small`, `text-embedding-ada-002` |
+| `$ai_provider` | The LLM provider<br/>Example: `openai`, `cohere`, `voyage` |
+| `$ai_input` | The text to embed<br/>Example: `"Tell me a fun fact about hedgehogs"` or array of strings for batch embeddings |
+| `$ai_input_tokens` | The number of tokens in the input |
+| `$ai_latency` | **Optional**<br/>The latency of the embedding call in seconds |
+| `$ai_http_status` | **Optional**<br/>The HTTP status code of the response |
+| `$ai_base_url` | **Optional**<br/>The base URL of the LLM provider<br/>Example: `https://api.openai.com/v1` |
+| `$ai_request_url` | **Optional**<br/>The full URL of the request made to the embedding API<br/>Example: `https://api.openai.com/v1/embeddings` |
+| `$ai_is_error` | **Optional**<br/>Boolean to indicate if the request was an error |
+| `$ai_error` | **Optional**<br/>The error message or object if the embedding failed |
+| **Cost Properties** | *Optional - If not provided, costs will be calculated automatically from token counts* |
```

Comment:
I liked the whole row as a header thing until this bit.

Maybe we just break up this table with headings and a small description for all of these? They're a little hard to read when you need to add additional info like here

Reply:
Yes, lets just do new tables, I think it will look better.